### ISSUE TITLE
I keep getting this error

### ISSUE THREAD

**Issue by Zyaxsor** [u3dreal/molecular-plus#17]
![image_2021-12-31_014609](https://user-images.githubusercontent.com/70926635/147808094-aec1a448-d6b2-47e9-8679-fafb46303051.png)

**Comment by Zyaxsor** [comment]
wont let me enable the addon I've tried everything, any help would be great, this beautiful addon is something I NEED

**Comment by u3dreal** [comment]
Use 3.0 Releae not alpha-master
OS ??
Molecular Version ??

**Comment by Zyaxsor** [comment]
I originally tried with 3.0 but thought since every example i find uses the alpha I should give that a go

![image](https://user-images.githubusercontent.com/70926635/147832830-94387f9a-b9a4-46b4-8a51-297ee496a998.png)

still to no avail either way, I also tried using 2.93 with its respectable addon version of 1.12 which didnt work for me either but I'm starting to think it's a problem exclusive to my pc

My current OS is Window 11 pro        
Version: 10.0.22000 Build 22000

I tried all different molecular versions and still nothing

I tried again but after completely uninstalling blender and reinstalling and have put in hours into trying to solve this myself trying anything and everything I can think of including messing around with the python code that I know very little of to fix in the program (and when it doesn't work re-installing blender [including deleting appdata files] because those files were of course now compromised). it is now day 2.... as I'm writing this, I have been granted a miracle and I cant believe I have finally gotten it to work using blender 3.0 and the specific "molecular-plus_1.1.7_39_win.zip" no other latest versions from you or other molecular contributors worked. I'm honestly so relieved and happy to be able to finally use this addon. 

Thank you so much for your contribution and for offering your assistance. Seriously, it is really appreciated and blender really should've incorporated this into the software a long time ago because blender NEEEDS this

thanks again <3
